### DISCUSSION TITLE
法律大模型--法内狂徒队（已满）

### DISCUSSION THREAD

**Discussion by JakChen797** [InternLM/Tutorial#345] [project]
### 为什么要做法律模型

法律大模型相对普通的chat模型，说话的语气需要更加专业，且需要有显著正确的法律知识作为支撑。因此：

+ 在专有的法律指令数据集上微调十分重要
+ 需要外挂专业知识库，通过RAG增强回答

### 项目核心

经过查找，目前已经有一些比较优秀的开源法律大模型。但一般只开源了权重文件，因此本项目的目的在于基于base模型（InternLM7B/20B），通过微调和RAG产出一个专业的法律大模型。

### 大概要做哪些事

+ 确定base模型，可能包括InternLM、Baichuan13B、LLaMA 7/13b等
+ 在法律指令数据集上做高效微调，并探索各种超参数设置
+ 通过LangChain外挂知识库，包括知识库的收集和建立，以及实现RAG
+ 量化部署模型，产生demo

### 项目成员
@sped-zhang 
@ZixinxinWang

**Comment by JakChen797** [comment]
本人是15班成员

**Comment by sped-zhang** [comment]
加我一个，我在7班

**Comment by leonfrank** [comment]
可以加上我吗

**Reply by JakChen797** [reply]
抱歉，人已经够了

**Comment by zhaoxin111** [comment]
项目还缺人吗？

**Reply by JakChen797** [reply]
抱歉，人已经够了
